### PR TITLE
Fix code issues

### PR DESCRIPTION
**Pull request by vkhramtsov**
It looks like  Scrutinizer does not check PR's anymore.

### CODE REVIEW

File: src/Captioning/Format/JsonFile.php [hunk 1]
```diff
@@ -20,16 +20,14 @@ public function parse()
...
-
-            $this->addCue($cue);
+        if (count($decodedContent['captions']) > 0) {
+            foreach ($decodedContent['captions'] as $c) {
+                $cue = new JsonCue($c['startTime'], $c['startTime'] + $c['duration'], $c['content']);
+                $cue->setDuration($c['duration']);
+                $cue->setStartOfParagraph($c['startOfParagraph']);
+
+                $this->addCue($cue);
+            }
```
Comment (delphiki) [inline src/Captioning/Format/JsonFile.php]: This adds complexity to the method.

File: src/Captioning/Format/SubripFile.php [hunk 2]
```diff
@@ -108,11 +114,7 @@ public function buildPart($_from, $_to)
             $cue = $this->getCue($j);
             $buffer .= $i.$this->lineEnding;
             $buffer .= $cue->getTimeCodeString().$this->lineEnding;
-            $buffer .= $cue->getText(
-                    $this->options['_stripTags'],
-                    $this->options['_stripBasic'],
-                    $this->options['_replacements']
-                );
+            $buffer .= $cue->getText();
```
Comment (delphiki) [inline src/Captioning/Format/SubripFile.php]: Why removing the parameters?

Comment (vkhramtsov) [inline src/Captioning/Format/SubripFile.php]: Sorry, my fault. Messed up with base class. I will revert it.

File: src/Captioning/Format/JsonFile.php (hunk 1 above)
Comment (vkhramtsov) [inline src/Captioning/Format/JsonFile.php]: I don't think so. Also it matches interface, because method should return `$this` instead of bool.

**Comment by vkhramtsov** [comment]
@delphiki, would you be so kind to check this PR once again.

File: src/Captioning/Format/JsonFile.php (hunk 1 above)
Comment (delphiki) [inline src/Captioning/Format/JsonFile.php]: Ah, good point.

**Comment by delphiki** [comment]
LGTM, thanks!

**Pull request merged**